ide-support/revsblibrary.livecodescript

```diff
@@ -1838,16 +1888,52 @@ command revSBUpdateDeployParams pStack, pStandaloneSettings, @xDeployParams
       appendToStringList tExtensionFile, tModules
    end repeat
    
+   local tLibraryStack
    repeat for each line tLibrary in pStandaloneSettings["scriptLibraries"]
       # AL-2015-03-05: [[ Bug 14813 ]] Transform script library display name to stack name at deploy time
-      local tLibraryStack
       put revSBStackNameFromScriptLibraryName(tLibrary) into tLibraryStack
       if there is a stack tLibraryStack then
          appendToStringList the effective filename of stack tLibraryStack, tAuxiliaryStackfiles
          appendToStringList "revInternal__LoadLibrary" && tLibraryStack, tStartupScript
       end if
    end repeat
    
+   local tLoadedLibraries
+   put revInternal__ListLoadedLibraries() into tLoadedLibraries
+   split tLoadedLibraries by return as set
+   
+   repeat for each line tLibrary in pStandaloneSettings["externals"]
+      -- check if there are any supporting libraries for the external and add
+      if char 1 of tLibrary is "!" then
+         local tExtPath
+         put revSBIncludedExternalPath(pPlatform, char 2 to -1 of tLibrary) into tExtPath
+         
+         if tExtPath is not empty then
+            set the itemDelimiter to slash
+            local tCurrentFolder
+            put the folder into tCurrentFolder
+            set the folder to item 1 to -2 of tExtPath
+            
+            local tFiles
+            put the files into tFiles
+            filter tFiles with "*.livecode*"
+            
+            repeat for each line tLibraryStack in tFiles
+               if there is a stack tLibraryStack then
+                  local tStackName
+                  put the short name of stack tLibraryStack into tStackName
+                  -- if it is not currently loaded then don't add as inclusion
+                  if tLoadedLibraries[tStackName] then
+                     appendToStringList the effective filename of stack tLibraryStack, tAuxiliaryStackfiles
+                     appendToStringList "start using stack" && quote & tStackName & quote, tStartupScript
+                  end if
```

Comment:
ok... just wanted to make sure I was following where you were going with this :-)